packages/vsx-registry/src/browser/vsx-extensions-contribution.ts

```diff
@@ -225,7 +225,9 @@ export class VSXExtensionsContribution extends AbstractViewContribution<VSXExten
             });
         });
         const selectedItem = await this.quickInput.showQuickPick(items, {
-            placeholder: nls.localizeByDefault('Select Version to Install'), runIfSingle: false
+            placeholder: nls.localizeByDefault('Select Version to Install'),
+            runIfSingle: false,
+            activeItem: items.find(item => item.label === currentVersion)
```

Comment:
An alternative would be `let activeItem = undefined` and then set it around line 219 when we discover the item that matches the current version.